public/css/main.css

```diff
@@ -0,0 +1,934 @@
+  padding: 2em 2em 1em 2em;
+}
+p {
+  line-height: 1.5;
+  letter-spacing: 0.02em;
+}
+a {
+  text-decoration: none;
+}
+button {
+  border: none;
+  background-color: #fd6632;
+  color: white;
+  padding: 1em 1.5em;
+  font-size: 1em;
+}
+button:hover {
+  cursor: pointer;
+}
+article h1, .services h3, .about h2, .clients h2 {
+  position: relative;
+  font-size: 1.777em;
+  overflow: hidden;
+  text-align: center;
+  letter-spacing: 0.1em;
+}
+.flex-container {
+  /*adapted from flexboxin5.com*/
+  display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
+  display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
+  display: -ms-flexbox;      /* TWEENER - IE 10 */
+  display: -webkit-flex;     /* NEW - Chrome */
+  display: flex;             /* NEW, Spec - Opera 12.1, Firefox 20+ */
+}
+.nav ul {
+  list-style-type: none;
+}
+.nav a {
+  color: rgb(124,131,135);
+}
+.nav a:hover {
+  color: rgb(0,0,0);
+}
+/* General styles desktop layout */
+@media screen and (min-width: 50em) {
+  body {
+    max-width: 80em;
+    width: 90%;
+    margin: 0 auto;
+  }
+  article {
+    padding: 2em 8%;
+  }
+  article h1, .services h3, .about h2, .clients h2 {
+    font-size: 2.369em;
+  }
+}
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    Mock strikethrough styles
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+article h1:before, article h1:after, .services h3:before, .services h3:after, .about h2:before, .about h2:after, .clients h2:before, .clients h2:after {
+  position: absolute;
+  top: 51%;
+  overflow: hidden;
+  width: 50%;
+  height: 1px;
+  content: '\a0';
+  background-color: #fff;
+}
+.portfolio h1:before, .portfolio h1:after, .team h1:before, .team h1:after, .clients h2:before, .clients h2:after {
+  background-color: rgb(124,131,135);
+}
+article h1:before, .services h3:before, .about h2:before, .team h1:before, .clients h2:before {
+  margin-left: -50%;
+  text-align: right;
+}
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    Bottom-notch styles
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+.header-notch:after, .header-notch:before {
+  position: absolute;
+  content: "";
+  width: calc(50% - 88px);
+  height: 0;
+  border-bottom: 20px solid #fd6632;
+  border-left: 85px solid transparent;
+  left: calc(50% + 3px);
+  bottom: 0;
+}
+.header-notch:before {
+  border-left: none;
+  border-right: 85px solid transparent;
+  left: 0;
+}
+.orange-notch, .white-notch, .black-notch {
+  position: relative;
+  left: 50%;
+  bottom: 16px;
+}
+.orange-notch:before,
+.orange-notch:after, .white-notch:before,
+.white-notch:after, .black-notch:before,
+.black-notch:after {
+    position: absolute;
+    content: "";
+    left: calc(50% + 10px);
+    bottom: -41px;
+    width: 40px;
+    height: 100px;
+    background-color: #fd6632;
+    -moz-border-radius: 0 180px 50px 50px;
+    border-radius: 0 100px 50px 50px;
+    -webkit-transform: rotate(-75deg);
+       -moz-transform: rotate(-75deg);
+        -ms-transform: rotate(-75deg);
+         -o-transform: rotate(-75deg);
```

Comment:
The latest versions of Firefox and Opera no longer need vendor prefixes. `-moz-` and `-o-` can and should be omitted. Be sure to verify your vendor prefixes with http://caniuse.com/#feat=transforms2d

I often use http://pleeease.io/play/ as a quick test for the latest required vendor prefixes for a given property. It's just a quick reference; Can I Use should be the authoritative source.